WORKSPACE

```diff
@@ -1,10 +1,9 @@
-'''
+"""
 This file lists and imports all external dependencies needed to build Oppia Android.
-'''
+"""
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
```

Comment:
Linter says `jvm_maven_import_external ` this is not coming use. 
Are we good to remove this or should we use some disable tag to avoid such auto-deletion by linter?

```
WORKSPACE:6: load: Loaded symbol "jvm_maven_import_external" is unused. Please remove it.
To disable the warning, add '@unused' in a comment. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#load)
```

Reply:
Unused symbols should be safe to remove. If the Bazel targets build, then we're good.

Reply:
Okay